test/integration/global-search.v2.test.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable no-console */
+/**
+ * Copyright 2020 IBM All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+
+// Set up the environment
+require('./setUpHelper.js');
+const GlobalSearchV2 = require('../../dist/global-search/v2');
+const {IamAuthenticator} = require('ibm-cloud-sdk-core');
```

Comment:
Not a requirement but stylistically this should be:

```suggestion
const { IamAuthenticator } = require('ibm-cloud-sdk-core');
```